player/lua/osc.lua

```diff
@@ -2353,6 +2442,11 @@ mp.set_key_bindings({
 }, "input", "force")
 mp.enable_key_bindings("input")
 
+mp.set_key_bindings({
+    {"mbtn_left",           function(e) process_event("mbtn_left", "up") end,
+                            function(e) process_event("mbtn_left", "down")  end},
+}, "window-controls", "force")
+mp.enable_key_bindings("window-controls")
```

Comment:
Can't you add this to the existing key bindings stuff?

Reply:
So, as I understand the code, and from my testing, each defined OSC area needs a separate input binding list. ie: The bindings only get applied to events detected inside the OSC area, and these are matched by name.

So, to share the main input binding list, would require defining the "input" area over the whole screen, to cover the window controls too.